packages/@aws-cdk/aws-backup/lib/backupable-resources-collector.ts

```diff
@@ -41,9 +41,21 @@ export class BackupableResourcesCollector implements IAspect {
     }
 
     if (node instanceof rds.CfnDBInstance) {
+      const dbInstance = node as rds.CfnDBInstance;
+      if (!dbInstance.dbClusterIdentifier) {
+        this.resources.push(Stack.of(node).formatArn({
+          service: 'rds',
+          resource: 'db',
+          arnFormat: ArnFormat.COLON_RESOURCE_NAME,
+          resourceName: node.ref,
+        }));
+      }
+    }
```

Comment:
If this change is correct: Can we just change it? Or do we have to treat it as a breaking change and create a feature flag? (I feel more confident when we create a feature flag.)